tools/internal_ci/linux/grpc_distribtests_standalone.cfg

```diff
@@ -15,8 +15,8 @@
 # Config file for the internal CI (in protobuf text format)
 
 # Location of the continuous shell script in repository.
-build_file: "grpc/tools/internal_ci/linux/grpc_distribtests_standalone.sh"
-timeout_mins: 120
+build_file: "grpc/tools/internal_ci/linux/grpc_distribtests_python.sh"
+timeout_mins: 360
```

Comment:
same question here regarding 6hrs

Reply:
This change is just a trick to "kidnap" an existing PR job to be able to get the test results. (It's the "DO NOT MERGE" commit in the commit history). I will remove this commit before merging.